src/viur/core/modules/file.py

```diff
@@ -392,6 +392,31 @@ class FileLeafSkel(TreeSkel):
         }
     )
 
+    @classmethod
+    def __inject_serving_url(cls, skel: SkeletonInstance) -> None:
```

Comment:
```suggestion
    def _inject_serving_url(cls, skel: SkeletonInstance) -> None:
```

The name mangling make is uncomfortable if you are subclassing things (Access to `Skeleton._Skeleton__seo_key_trans` is already enough anoying for me ...). I would avoid it unless you have a real reason to keep things inside the same class and not make them available to sub or other classes. "Private" should be enough to signal internal use.